### ISSUE TITLE
Keeping Backspace pressed doesn't delete all the text

### ISSUE THREAD

**Issue by daniloercoli** [wordpress-mobile/AztecEditor-Android#853]
### Expected
To be able to keep Backspace pressed on the virtual keyboard and be able to keep deleing text.

### Observed
See the video on my nexus 5x: https://cloudup.com/cPoCbHUTMr4

- On a Nexus 5x with `Google Keyboard`
The caret stops almost immediately and I see the following in the log
```
2019-10-02 17:08:16.356 30234-30234/org.wordpress.aztec W/IInputConnectionWrapper: beginBatchEdit on inactive InputConnection
2019-10-02 17:08:16.356 30234-30234/org.wordpress.aztec W/IInputConnectionWrapper: beginBatchEdit on inactive InputConnection
2019-10-02 17:08:16.356 30234-30234/org.wordpress.aztec W/IInputConnectionWrapper: finishComposingText on inactive InputConnection
2019-10-02 17:08:16.356 30234-30234/org.wordpress.aztec W/IInputConnectionWrapper: deleteSurroundingText on inactive InputConnection
2019-10-02 17:08:16.356 30234-30234/org.wordpress.aztec W/IInputConnectionWrapper: endBatchEdit on inactive InputConnection
2019-10-02 17:08:16.356 30234-30234/org.wordpress.aztec W/IInputConnectionWrapper: setComposingRegion on inactive InputConnection
2019-10-02 17:08:16.356 30234-30234/org.wordpress.aztec W/IInputConnectionWrapper: endBatchEdit on inactive InputConnection
2019-10-02 17:08:16.356 30234-30234/org.wordpress.aztec W/IInputConnectionWrapper: beginBatchEdit on inactive InputConnection
2019-10-02 17:08:16.357 30234-30234/org.wordpress.aztec W/IInputConnectionWrapper: setComposingText on inactive InputConnection

```

- On a Samsung S9 with `Samsung Keyboard`
The caret stops at the end of the previous line and `sendKeyEvent on inactive InputConnection` is available in the log. No video available atm, but I can make it.
```
2019-10-02 16:53:48.260 28980-28980/org.wordpress.aztec D/InputMethodManager: prepareNavigationBarInfo() DecorView@ef89c9a[MainActivity]
2019-10-02 16:53:48.260 28980-28980/org.wordpress.aztec D/InputMethodManager: getNavigationBarColor() -855310
2019-10-02 16:53:48.260 28980-28980/org.wordpress.aztec V/InputMethodManager: Starting input: tba=org.wordpress.aztec ic=com.android.internal.widget.EditableInputConnection@791cfca mNaviBarColor -855310 mIsGetNaviBarColorSuccess true , NavVisible : true , NavTrans : false
2019-10-02 16:53:48.260 28980-28980/org.wordpress.aztec V/InputMethodManager: getMaxLengthForEditText InputFilter = java.lang.NoSuchMethodException: org.wordpress.aztec.AztecText$setupKeyListenersAndInputFilters$emptyEditTextBackspaceDetector$1.getMaxLength []
2019-10-02 16:53:48.260 28980-28980/org.wordpress.aztec D/InputMethodManager: startInputInner - Id : 0
2019-10-02 16:53:48.267 28980-28980/org.wordpress.aztec I/InputMethodManager: startInputInner - mService.startInputOrWindowGainedFocus
2019-10-02 16:53:48.270 28980-28980/org.wordpress.aztec D/InputTransport: Input channel constructed: fd=75
2019-10-02 16:53:48.270 28980-28980/org.wordpress.aztec D/InputTransport: Input channel destroyed: fd=76
2019-10-02 16:53:48.298 28980-28980/org.wordpress.aztec W/IInputConnectionWrapper: sendKeyEvent on inactive InputConnection
2019-10-02 16:53:48.298 28980-28980/org.wordpress.aztec W/IInputConnectionWrapper: sendKeyEvent on inactive InputConnection
```

**Comment by daniloercoli** [comment]
I tested the versions `v1.3.27`, and `v1.3.28` and both worked without problems.
The issue was introduced in `v1.3.29`. cc @marecar3

**Comment by marecar3** [comment]
Thanks, @daniloercoli, probably one of chromebook fixes introduced that bug.
I will come back with the details. cc @khaykov

**Comment by daniloercoli** [comment]
Addressed in #855.

**Comment by designsimply** [comment]
fwiw, I did a quick test of this on my Pixel 3 with Android 10 and saw the problem once in several tries. ([1m56s](https://cloudup.com/cjlekTjFMKa))
